components/x-gift-article/src/Form.jsx

```diff
@@ -12,20 +12,19 @@ export default (props) => (
 			<div role="group" arialabelledby="gift-article-title">
 				<Title {...props} />
 
-				{!props.isFreeArticle && (
-					<RadioButtonsSection
-						shareType={props.shareType}
-						isArticleSharingUxUpdates={props.isArticleSharingUxUpdates}
-						showGiftUrlSection={props.actions.showGiftUrlSection}
-						showEnterpriseUrlSection={props.actions.showEnterpriseUrlSection}
-						showNonGiftUrlSection={props.actions.showNonGiftUrlSection}
-						enterpriseLimit={props.enterpriseLimit}
-						enterpriseHasCredits={props.enterpriseHasCredits}
-						enterpriseRequestAccess={props.enterpriseRequestAccess}
-						enterpriseAlert={!props.enterpriseHasCredits && !props.enterpriseRequestAccess}
-						enterpriseEnabled={props.enterpriseEnabled}
-					/>
-				)}
+				<RadioButtonsSection
+					shareType={props.shareType}
```

Comment:
(Out-of-scope for this PR). The usage of props without PropTypes will easily lead to zombie properties on refactoring like this. Better would be to explicitly declare all the props as function parameters as done in the components.

Reply:
Agreed. This should be applied repo-wide though!